### ISSUE TITLE
Allow deploy_updates cloud hook function to accept a site parameter

### ISSUE THREAD

**Issue by bobbygryzynger** [acquia/blt#1718]
My system information:
* BLT version: `8.8.4`

This is a feature request. Often, when performing a database copy on ACE it's helpful to reimport configuration and run database updates against a branch that is being tested on a non-production environment. Much of the behavior provided by the `deploy_updates` bash function is useful in the `post-db-copy` context as well. The one thing lacking in this function is the ability to only run updates on a specific multisite — the site that received the database copy.

I would propose that `deploy_updates` (more specifically `ace_deploy`) accept a `$site_name` variable that would be used like this when it is defined:
```bash
blt deploy:update -Denvironment=$target_env -Dblt.verbose=true -Dmultisite.name=$site_name
```

With this usage, it would be up to the user to define the `$site_name` variable and if it is not defined the command would be run without `multisite.name`, as it currently is.

I'm happy to put together a PR for both `8.8.x` and `8.9.x` if this change seems worthwhile.

**Comment by bobbygryzynger** [comment]
Another thought I had here is that it might be nice if BLT provided an initialization function for cloud hooks along the lines of:
```sh
function blt_init() {
  repo_root="/var/www/html/$site.$target_env"
  export PATH=$repo_root/vendor/bin:$PATH
  cd $repo_root
}
```

Since this is necessary to run any BLT commands remotely.

All `hooks/common` scripts could be updated to include these lines so that projects could call BLT tasks in all cloud hooks out-of-the-box.
```sh
. /var/www/html/$site.$target_env/vendor/acquia/blt/scripts/cloud-hooks/functions.sh
blt_init
# hook-specific code...
```
This would ease running BLT during cloud hook scripts (and would eliminate some existing repetition).

**Comment by grasmash** [comment]
I think that you can already accomplish this with BLT 8.9.x  if you specify `-D site=$site_name`.

**Comment by grasmash** [comment]
Furthermore, you can already accomplish this on a per environment basis by creating a `blt/[env].yml`  file containing:

```
site: [site]
``

Where [site] is the name of the multisite and [env]  if the machine name of the Acquia environment.

**Comment by grasmash** [comment]
As for `blt_init()`,  I think that sounds like a nice idea.

**Comment by grasmash** [comment]
I am going to close this since it is effectively already an available feature on 8.9.X, and 8.8.X is in LTS status and not receiving new features at this time.
